v2/emailpassword/serverless/with-aws-lambda/backend-config.mdx

```diff
@@ -4,20 +4,72 @@ title: 2. Backend config
 hide_title: true
 ---
 
+<!-- COPY SECTION -->
+<!-- ./emailpassword/serverless/with-aws-lambda/backend-config.mdx -->
+<!-- 1 -->
+
 import AppInfoForm from "/src/components/appInfoForm"
 import CoreInjector from "/src/components/coreInjector"
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs";
+import TabItem from '@theme/TabItem';
 
 # 2. Backend config
 
-## 1) Install supertokens package
+## 1) Create Lambda layer for required libraries
+
+<BackendSDKTabs>
+<TabItem value="nodejs">
+
 ```bash
-npm i supertokens-node
+mkdir lambda && cd lambda
+touch lambdaFunction.js
+
+# Run these commands
+npm i -s supertokens-node @middy/core @middy/http-cors
+cp -r node_modules nodejs # TODO: Verify this
+zip -r supertokens-layer.zip nodejs/
 ```
 
-## 2) Create a configuration file (`config.js`)
-- Create a `config.js` file in the root directory of your project.
-- An example of this file can be found [here](https://github.com/supertokens/supertokens-auth-react/blob/master/examples/with-aws-lambda/backend/config.js).
+</TabItem>
+<TabItem value="python">
 
+```bash
+# Run these commands:  
+mkdir lambda && cd lambda
+touch lambda_function.py requirements.txt
+
+# TODO: Add lambda_function.py content for boilerplate:
+
+# Create requirements.txt file with the following content:
+fastapi
+uvicorn
+mangum
+nest_asyncio
+supertokens-python
```

Comment:
Give thee command to add these to the requirements.txt file. IUt's weird to have commentary in a bash script which the user is supposed to execute.

Reply:
Added a one liner command instead.